locales/es-ES.yml

```diff
@@ -0,0 +1,216 @@
+es-ES:
+
+  # global, logged out, and auth-related
+  add: Crear
+  add_new: Crear nuevo
+  already_have_account: Tengo una cuenta # signup form cta
+  back: Atrás
+  buy_now: Comprar
+  cancel: Cancelar
+  change_password: Cambiar Contraseña
+  charge_level: Nivel de Batería
+  charged: Cargada
+  close: Cerrar
+  confirm_change_password: Cambiar mi contraseña
+  confirm_password: Confirmar nueva contraseña
+  create_an_account: Crear cuenta de usuario # login form cta
+  days: días
+  delete: Borrar
+  device_id: Identificador de dispositivo
+  edit: Editar
+  email_address: Correo electrónico
+  first_name: Nombre
+  forgot_password: Olvidé mi contraseña # login form link
+  forgot_password_title: Otra vez # heading on /password/new (hint: https://www.google.com/search?q=here+we+go+again+meme)
+  get_started: ¡Empecemos!
+  identify: Identificar
+  keep_me_signed_in: Mantenerme conectado
+  knowledge_base: Base de Conocimiento
+  last_name: Apellido
+  login: Iniciar sesión
+  log_out: Cerrar sesión
+  minimum_characters: Caracteres como mínimo # ex: 8 characters minimum
+  mins: mins # 'minutes' abbreviation
+  new_password: Nueva contraseña
+  password: Contraseña
+  reset_password: Restablecer mi contraseña
+  sign_in: Iniciar sesión # login form cta
+  sign_up: Registrarse # logged out navigation
+  save: Guardar
+  update: Actualizar
+  view: Ver
+  welcome: ¡Hola!
```

Comment:
Spanish is a gendered language; most words have a he/she variant. A literal translation for welcome (Bienvenido) would be (he)Bienvenido or (she)Bienvenida. To avoid dated language, I've chosen a more liberal "Hi"